src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1852,13 +1837,85 @@
     </Field>
     <Field Name="Value" Type="ExpressionSyntax"/>
   </Node>
+
+  <AbstractNode Name="VariableComponentSyntax" Base="CSharpSyntaxNode">
+  </AbstractNode>
+  <Node Name="TypedVariableComponentSyntax" Base="VariableComponentSyntax">
+    <Kind Name="TypedVariableComponent"/>
+    <Field Name="Type" Type="TypeSyntax"/>
+    <Field Name="Designation" Type="VariableDesignationSyntax"/>
+  </Node>
+  <Node Name="TupleDeconstructionVariableComponentSyntax" Base="VariableComponentSyntax">
```

Comment:
How about this?
`TypedVariableComponentSyntax` renamed to `VariableComponentTypeFormSyntax`
`TupleDeconstructionVariableComponentSyntax` renamed to `VariableComponentListFormSyntax`

---
In reply to: [73585280](https://github.com/dotnet/roslyn/pull/12917#discussion_r73585280) [](ancestors = 73585280,73565725)